documentation/versioned_docs/version-v0.15/modules/authentication/tutorials/OAuth2/github.mdx

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_label: 'Github'
+---
+
+# Github
+
+In this tutorial, we'll be setting up GitHub authentication for Conduit.<br />
+
+## Prerequisites
+
+Before we begin, you should have a GitHub account and a Conduit application set up. If you don't, head over to the [Getting Started](../../get-started) section to get set up.
+
+## Setting up GitHub OAuth
+
+To set up GitHub OAuth, you'll need to create a new OAuth application on GitHub. You can do this by going to [GitHub's developer settings](https://github.com/settings/applications/new) and clicking the "New OAuth App" button.
+
+
+You'll need to fill out the form with the following information:
+
+- **Application name**: This is the name of your application. It can be anything you want.
```

Comment:
The 'It can be anything you want' phrase is repeated too much, I would delete it